JAY.txt

```diff
@@ -0,0 +1,72 @@
+import csv
+with open ("C:\\22BCA142\\python\\product_selling.csv",'w') as g:
+    data=csv.writer(g)
+    header=["prod_no","prod_name","jan","feb","mar","apr","may","jun"]
+    data.writerow(header)
+    for i in range(12):
+        prod_no=int(input("Enter product no:"))
+        prod_name=input("Enter name:")
+        jan=int(input("Enter product selling of jan:"))
+        feb=int(input("Enter product selling of feb:"))
+        mar=int(input("Enter product selling of mar:"))
+        apr=int(input("Enter product selling of apr:"))
+        may=int(input("Enter product selling of may:"))
+        jun=int(input("Enter product selling of jun:"))
+        record=[prod_no,prod_name,jan,feb,mar,apr,may,jun]
+        data.writerow(record)
+
+with open("C:\\22BCA142\\python\\product_selling.csv",'r')as g1:
+    content=csv.reader(g1)
+    for i in content:
+        print(i)
+2.Create dataframe.        
+import pandas as pd
+df=pd.read_csv("C:\\22BCA142\\python\\product_selling.csv")
+print(df)
+
+3. Change Column Name Product No, Product Name, January, February, March, April, May, June.
+df.columns=["Product_no","Product_name","January","February","March","April","May","June"]
+print(df)
+
+4.  Add column "Total Sell" to count total of all month and "Average Sell"

```

Comment:
how work this line

Reply:
this code is work for add total seel column and count average sell